addons/account/static/src/js/account_portal.js

```diff
@@ -1,6 +1,7 @@
 /** @odoo-module */
 
 import publicWidget from 'web.public.widget';
+import "portal.portal"; // force dependencies
```

Comment:
> However, in some cases, PortalHomeCounters is not yet loaded when those patches are to be applied.

Not sure what you mean: if portal.portal was not yet loaded when arriving here, there would a crash right?  (maybe that's what you meant and did not actually see a crash ever and it is just theoretical?). In any case, yes, it is of course better like that. Make that in its own PR for the right stable version (and remove it from here) (always do that actually, I don't see how this is related to the public widget / widget merge).

Also: don't import via `"portal.portal"` but with the new way (path to the file or something like that), the goal is to not use module aliases which are there for compatibility.

Reply:
This branch crashes without this commit. I guess it's because I move some imports around.

I did move it to 16.0 here: https://github.com/odoo/odoo/pull/125798